src/colour/govuk-blue/index.md

```diff
@@ -26,14 +26,14 @@ Accent teal also sits alongside to add impact and help the brand feel more moder
 </div>
 <div>
 
-![TODO](./wordmark-on-blue.svg)
+![GOV.UK wordmark shown on blue background.](./wordmark-on-blue.svg)
```

Comment:
(Optional) Because this section is about colours and because Primary blue and Accent teal are mentioned just before this, I think it's worth calling out that the blue is not any blue but Primary blue and that the dot is in Accent teal.